### ISSUE TITLE
[BUG] Kirby Link does not render icon on web

### ISSUE THREAD

**Issue by askarby** [kirbydesign/designsystem#1735]
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the bug
When creating a Kirby Link (that should open in "a new tab"), an icon well be presented right of the link text (see: https://cookbook.kirby.design/home/showcase/link)

This works absolutely fine, when the application is hosted on a root, domain (either being http://this-is-my-domain-with-an-app-using-kirby.com/ or http://localhost:4200) - but when it's hosted on a sub-folder level, the icon is not rendered!

## Describe how to reproduce the bug

1. Clone a version of Kirby (`git clone git@github.com:kirbydesign/designsystem.git`)
2. Edit the `apps/cookbook/src/index.html`'s base href to be `/cookbook/` 
3. Build kirby's cookbook (`npm run dist`)
4. Launch http-server, to make Kirby's cookbook available in a sub-folder (`npx http-server dist`)
5. Open Link-page in a browser (http://localhost:8080/cookbook/home/showcase/link)
6. Now see that the icon is gone!

## Which Kirby version was used?
3.0.8

## What was the expected behavior?
The icon is rendered regardless of which sub-folder the application is served from!

## Add any screenshots
![image](https://user-images.githubusercontent.com/44014/132311650-3b5d717a-a0d4-4524-8020-23cc7864c108.png)

## Please complete the following information:
- OS: MacOS
- Browser Chrome
- Browser version 93

## Are there any additional context?
Our CI / staging environment run's on a singular Azure host, that utilizes sub-folders for hosting "a build per pull request". It's quite annoying having to repeatedly explain that this issue is not present in production.

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](../CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](../SUPPORT.md/#team-kirby); do not proceed until this is done

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](../CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [master branch](https://github.com/kirbydesign/designsystem/tree/master) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.

**Comment by RasmusKjeldgaard** [comment]
Duplicate of https://github.com/kirbydesign/designsystem/issues/1565
